S3TransferUtilitySample/AndroidManifest.xml

```diff
@@ -3,10 +3,6 @@
     android:versionCode="1"
     android:versionName="1.0" >
 
-    <uses-sdk
```

Comment:
Why is this being removed?

Reply:
Android Studio with latest gradle plugin does not allow declaring minSDK version in manifest file. Gives the following error:
"The minSdk version should not be declared in the android manifest file. You can move the version from the manifest to the defaultConfig in the build.gradle file."